tests/errata/test_errata_metadata.py

```diff
@@ -19,7 +19,7 @@ def test_errata_typical_metadata(fake_errata_tool):
     # It should have queried the expected XML-RPC endpoint.
     # Note that our https was replaced with http, this is expected!
```

Comment:
This comment is now outdated, this will probably confuse a casual reader.  Could you please adjust or remove it?  Note: the same comment appears in a few places.